libraries/consul_config.rb

```diff
@@ -69,10 +69,13 @@ class ConsulConfig < Chef::Resource
       attribute(:key_file, kind_of: String)
       attribute(:leave_on_terminate, equal_to: [true, false], default: false)
       attribute(:log_level, equal_to: %w(INFO DEBUG WARN), default: 'INFO')
+      attribute(:max_stale, kind_of: String)
```

Comment:
The documentation says that `max_stale` is a subset of `dns_config`, not a standalone option.
https://www.consul.io/docs/agent/options.html#max_stale